cli/cmd/logs.go

```diff
@@ -97,10 +99,36 @@ func (o *logsOptions) toSternConfig(controlPlaneComponents, availableContainers
 	if err != nil {
 		return nil, err
 	}
+
+	// Based on stern/cmd/cli.go
```

Comment:
It's unfortunate that this fix has not been merged to stern yet :(. The more we patch this the more I feel like we should just implement this feature ourselves.

Reply:
should this comment link to a github issue in the upstream stern repo?  I don't know what this is fixing.

Reply:
this particular comment is not a fix, simply borrowing code from stern, and i wanted to call it out.

this does bring up a good point around the fixes that were necessary to workaround the log line interleaving issue. i've updated the PR with a comment around that part of the code.